package-lock.json

```diff
@@ -3307,28 +3307,28 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": false,
```

Comment:
Can we exclude changes in this file since no dependencies were changed?

Reply:
Done